ff/serving/provider/snowflake.go

```diff
@@ -68,3 +68,7 @@ func snowflakeOfflineStoreFactory(config SerializedConfig) (Provider, error) {
 	}
 	return store, nil
 }
+
+func (q snowflakeSQLQueries) materializationDrop(tableName string) string {
```

Comment:
is this syntax unique to snowflake?

Reply:
I've left the default as dropping a materialized view. Snowflake just uses a table in this case so I made it non-default